### PR TITLE
ALARMS: Add odom-kill for time and jumps

### PR DESCRIPTION
**Pull request by LucasBA**
Logic is the same as what exists in Navigator

### CODE REVIEW

File: command/sub8_alarm/alarm_handlers/odom_kill.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        self.last_position = msg.pose.pose.position
+        self.last_time = rospy.Time.now()
+
+    def check_continuity(self, new_odom_msg):  # True if 'continuous'
+        if self.odom_discontinuity:
+            return
+
+        this_p = new_odom_msg.pose.pose.position
+        last_p = self.last_position
+        jump = ((this_p.x - last_p.x) ** 2 + (this_p.y - last_p.y) ** 2) ** 0.5
```
Comment (mattlangford) [inline command/sub8_alarm/alarm_handlers/odom_kill.py]: We in 3D now, might want to change this to reflect that

**Review by mattlangford — APPROVED** [review]
If this is just a copy paste from Navigator you should probably give David credit

**Review by DSsoto — APPROVED** [review]
Looks good.

File: command/sub8_alarm/alarm_handlers/odom_kill.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def check_continuity(self, new_odom_msg):  # True if 'continuous'
 
         this_p = new_odom_msg.pose.pose.position
         last_p = self.last_position
-        jump = ((this_p.x - last_p.x) ** 2 + (this_p.y - last_p.y) ** 2) ** 0.5
+        jump = ((this_p.x - last_p.x) ** 2 + (this_p.y - last_p.y) ** 2 + (this_p.z - last_p.z) ** 2) ** 0.5
```
Comment (DSsoto) [inline command/sub8_alarm/alarm_handlers/odom_kill.py]: I forgot we operate in more dimensions.

**Pull request merged**